main.go

```diff
@@ -33,10 +39,86 @@ func main() {
 	sync := marathon.NewMarathonSync(remote, consul)
 	go sync.Sync()
 
-	// set up routes
+	fh := &ForwardHandler{consul}
+
+	version, err := remote.Version()
+	if version >= "0.9.0" {
```

Comment:
Yeah, this is not good. How would you feel about pulling in an actual version number comparison library, e.g. https://github.com/hashicorp/go-version ?